### ISSUE TITLE
shortcut for Path.resolve

### ISSUE THREAD

**Issue by robstoll** [robstoll/atrium#216]
*Platform* (jvm, js, android): jvm
*Extension* (none, kotlin 1.3, jdk8): jdk8

## Code related feature
```
expect(Paths.get("folder")).resolve("file.txt").exists()
expect(Paths.get("folder")).resolve("file.txt") {
    isReadable()
    isWriteable()
}

//instead of

expect(Paths.get("folder")).feature { f<String, Path>(it::resolve, "file.txt") }.exists()
expect(Paths.get("folder")).feature({ f<String, Path>(it::resolve, "file.txt") }) {
    isReadable()
    isWriteable()
}
```

Following the things you need to do:

*lib*
- implement _resolve in PathAssertions by using `ExpectImpl.feature.f1`

*domain*
- extend PathAssertions with a function `resolve` (see existing functions as a guideline)
- modify PathAssertionsBuilder, delegate to PathAssertions (see existing functions as a guideline)
- delegate implementation to robstoll-lib in PathAssertionsImpl (see existing functions as a guideline)

*api*
- provide a fun which expects a String and returns Expect<Path> (see listAssertions.kt -> get as a guideline)
- provide a fun which expects a String plus an `assertionCreator`-lambda and returns Expect<T> where `T: Path` (see listAssertions.kt -> get as a guideline)
- add `@since 0.10.0` to KDOC
- add the function also to api-infix-en_GB-jdk8
- add your new functionality to PathFeatureAssertionsSpec in specs-common
  - extend it in atrium-api-fluent-en_GB-jdk8-jvm/src/test
  - extend it in atrium-api-infix-en_GB-jdk8-jvm/src/test

## Your first contribution?
- Write a comment `I'll work on this` if you would like to take this issue over. 
  This way we get the chance to revise the description in case things have changed in the meantime,
  we might give you additional hints and we can assign the task to you, so that others do not start as well.
- See [Your first code contribution](https://github.com/robstoll/atrium/blob/master/.github/CONTRIBUTING.md#your-first-code-contribution) for guidelines.  
- Do not hesitate to ask questions here or to contact us via [Atrium's slack channel](https://kotlinlang.slack.com/team/U3DE1TXKP) if you need help
  ([Invite yourself](https://slack.kotlinlang.org/) in case you do not have an account yet).

**Comment by Keerthi4308** [comment]
Hi, I like to work on this task

**Comment by robstoll** [comment]
@Keerthi4308 do you need help?

**Comment by aljacinto** [comment]
Hi @robstoll I'll work on this

**Comment by robstoll** [comment]
@aljacinto do you need any help?

**Comment by aljacinto** [comment]
Sorry @robstoll, got busy.  Can you reassign to someone else?

**Comment by slalu** [comment]
I will work on this!

**Comment by robstoll** [comment]
@slalu I updated the description, I am going to add a bit more description for the infix API later on.

**Comment by robstoll** [comment]
implemented with 9cb8c5de
